docs/source/blog/displacement-vectors.md

```diff
@@ -0,0 +1,47 @@
+---
+blogpost: true
+date: October 27, 2025
+author: Carlo Castoldi
+location: Milan, Italy
+category: update
+language: English
+---
+
+# Improving displacement vectors
```

Comment:
The title works, but I wonder if we can make it a bit more specific. I've listed some ideas below, let me know if you like any of them better than the current title (some of them are a bit cheeky):
- Redefining displacement vectors for greater clarity
- A fresh look at displacement vectors
- Displacement revisited

Just below the title, I would add a 1-sentence summary of the whole blogpost. The first paragraph is important because it will appear as a "previw" on the "blog" page that lists all the blogposts.

I'd suggest something like this:

> This post introduces recent improvements to how `movement` computes displacement vectors, making the definitions more explicit, flexible, and intuitive for users.